docs/tasks/administer-cluster/change-pv-reclaim-policy.md

```diff
@@ -0,0 +1,72 @@
+---
+---
+
+{% capture overview %}
+This page shows how to change the reclaim policy of a Kubernetes Persistent
+Volume.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
```

Comment:
(Correct me if I'm wrong, but) I don't think this is the right prerequisite. I don't think the reader could use Minikube with a PersistentVolume.

I think we probably need something similar to what we've done [here](http://kubernetes.io/docs/tutorials/stateless-application/expose-external-ip-address/).